pyslackers_website/marketing/views.py

```diff
@@ -27,3 +27,11 @@ def get_context_data(self, **kwargs):
 
 class TermsOfService(TemplateView):
     template_name = 'marketing/terms_of_service.html'
+
+
+def info(request):
+    app_version = os.getenv("app_version")
```

Comment:
Consistent quotes, we are using `'` in most places. I'd also rename the envvar to be more explicit and consistent with common envvar usage: `APP_GIT_REVISION`